src/sider/historikk/Historikk.tsx

```diff
@@ -86,7 +86,7 @@ export function Historikk({ historikkEvents, tilfeller }: Props): ReactElement {
   }
 
   return (
-    <div className="p-4">
+    <Box background="surface-default" className="p-4">
```

Comment:
Vet ikke om det er lettere å lese stylingen ved bruk av tailwind eller padding property-en i box elementet. 
```suggestion
    <Box background="surface-default" padding="4">
```